printer_mods/chirpy/NozzleWiper/README.md

```diff
@@ -0,0 +1,154 @@
+# Nozzle Wiper
+This is a nozzle purge and wipe mod for the Voron V0.1. It is based on the [SlideSwipe](https://github.com/chestwood96/SlideSwipe) magnetic probe by [chestwood96](https://github.com/chestwood96).
+
+It uses a snap in frame clip, that doesn't require screws, to attach a servo
+that extends a bucket and brush over the build plate for the hotend nozzle to
+clean it of filament debris for cleaner prints.
+
+![Nozzle Wiper](images/NozzleWiper.gif)
+
+## Goals:
+
+- Servo used to extend system over the print bed
+- Bucket and brush to clean nozzle
+- Snap in frame clip that does not need screws
+- Klipper macros to control the entire process
+
+These have all been implemented in this public release.
+
+## Printing:
+
+- Use the Voron defaults and print in ABS or better
+- The parts are orientated correctly in the STLs
+- There is a single All In One STL that fits onto the v0.1 build plate to make life easier
+
+## BOM:
+
+- 1x Micro Servo 9G Servo Motor (SG90/MG90). MG90 servo version is preferred to cope with enclosure temperatures
+- 2x M3x8mm BHCS (for servo mount)
+- 2x M3x6mm BHCS (for brush holder mount)
+- 2x M2x10mm self tapping screws (optional: for snap in frame clip)
+- 1x Nozzle cleaning brush in Copper
+- 3x OD 4mm, length 16mm, PTFE tube
+- 1x OD 4mm, length 7mm, PTFE tube
+
+## Hardware:
+
+Brush:
+https://www.amazon.co.uk/ALMOCN-Printer-Nozzle-Cleaning-Toothbrush/dp/B08XYMVNSG
+https://www.aliexpress.com/item/4000801101276.html?spm=a2g0o.productlist.0.0.548c6ad8feWJPk
```

Comment:
The tracking parameters should be removed from the aliexpress urls (eg. https://www.aliexpress.com/item/4000801101276.html)